documentation/assemblies/configuring/assembly-storage.adoc

```diff
@@ -13,11 +13,15 @@ The supported storage types are:
 * Ephemeral (Recommended for development only)
 * Persistent
 * JBOD (Kafka only; not available for ZooKeeper)
+* Tiered storage (Technology preview)
 
-To configure storage, you specify `storage` properties in the custom resource of the component. 
+To configure non-tiered storage, you specify `storage` properties in the custom resource of the component. 
 The storage type is set using the `storage.type` property.
 
-When using node pools, you can specify storage configuration unique to each node pool used in the cluster.
+Support for tiered storage is being introduced as a technology preview.
```

Comment:
Again, no technology preview. Just add warning that this is early access in Apache Kafka and has several limitations.